packages/admin-panel/src/pages/resources/DashboardRelationsPage.jsx

```diff
@@ -107,8 +107,9 @@ const FIELDS = [
 ];
 
 const CREATE_CONFIG = {
-  title: 'Create a new relation between Dashboard and DashboardItem',
```

Comment:
Potentially this older title might be useful in some form, because to a PM 'dashboard relation' might not be immediately obvious

Reply:
Switched back over—I was on the fence about which one to go with too